### PR TITLE
Add ADR on Github Actions usage

### PR DESCRIPTION
**Pull request by giorgiosironi**
Not meant for acceptance and going all-in yet - keen to understand how all our trials are going before committing to it.

### CODE REVIEW

**Comment by NuclearRedeye** [comment]
I've done some initial experimentation with building an Node.js package and publishing it to NPM. I was lead to do this by an issue I was having with Travis, which I couldn't debug easily. I was happy with actions so far, but would be keen to experiment more with debugging builds and environments.

**Comment by giorgiosironi** [comment]
From the Publisher side we have our [first full build](https://github.com/libero/article-store/actions). Notable points the artifact usage to build Docker images (or `node_modules` if you'd like) only once, parallel jobs and reduced code duplication using a matrix.

**Review by davidcmoulton — APPROVED** [review]
Approving this becoming proposed.

File: doc/adr/0002-continuously-delivery-travis-ci.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Date: 2018-10-09
 
 ## Status
 
-Accepted
+[Superseded](0007-github-actions.md)
```
Comment (thewilkybarkid) [inline doc/adr/0002-continuously-delivery-travis-ci.md]: Not until it's accepted?

Comment (giorgiosironi) [reply doc/adr/0002-continuously-delivery-travis-ci.md]: Yeah, I mixed up the status of this PR (draft/not ready) with the status of the ADR (still `Proposed` if it was to be merged).

**Review by diversemix — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Still one more :+1: needed

**Review by erezmus — APPROVED** [review]

**Pull request merged**